sample-apps/react-native/dogfood/src/components/LivestreamChat.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import { Platform, SafeAreaView, StyleSheet } from 'react-native';
+import {
+  Channel,
+  MessageInput,
+  MessageList,
+  useChatContext,
+} from 'stream-chat-react-native';
+import { AuthenticationProgress } from '../components/AuthenticatingProgress';
+import { Channel as ChannelType } from 'stream-chat';
+import { StreamChatGenerics } from '../../types';
+import { useBottomSheetInternal } from '@gorhom/bottom-sheet';
+import { appTheme } from '../theme';
+
+export type LivestreamChatProps = {
+  callId: string;
+  callType: string;
+};
+
+export const LivestreamChat = ({ callId, callType }: LivestreamChatProps) => {
+  const [channel, setChannel] = useState<
+    ChannelType<StreamChatGenerics> | undefined
+  >(undefined);
+  const { client } = useChatContext();
+  const { shouldHandleKeyboardEvents } = useBottomSheetInternal();
+
+  // Done as per https://ui.gorhom.dev/components/bottom-sheet/keyboard-handling/ to solve the issue around keyboard hiding the text input in the chat inside bottom sheet.
```

Comment:
the docs linked here doesnt have the code we have written here.. confusing to me

Reply:
https://github.com/gorhom/react-native-bottom-sheet/blob/master/src/components/bottomSheetTextInput/BottomSheetTextInput.tsx

I have followed the tip within these docs(https://ui.gorhom.dev/components/bottom-sheet/keyboard-handling/), which links to this component.

Reply:
![Screenshot_1712555407](https://github.com/GetStream/stream-video-js/assets/39884168/32e42d88-602b-41ad-a796-e69976b871d1)
This is what happens on Android if I apply this hack. Everything is normal without it.

Reply:
Done 👍🏻